lib/logger.js

```diff
@@ -374,6 +347,10 @@ class Logger {
     }
 
     close() {
+        if (this._logger && this._logger.removeListener) {
+                this._logger.removeListener('error', this._logger._errorHandler);
```

Comment:
It's not `this._logger._errorHandler`, it's `this._errorHandler`

Reply:
Good point